packages/core/src/dialects/abstract/query-interface.js

```diff
@@ -527,17 +519,13 @@ export class AbstractQueryInterface extends AbstractQueryInterfaceTypeScript {
       assertNoReservedBind(options.bind);
     }
 
-    options = cloneDeep(options) ?? {};
     if (typeof where === 'object') {
       where = cloneDeep(where) ?? {};
```

Comment:
we don't need to clone deep `where`, we don't mutate it

Reply:
Updated now.